examples/server/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM autofleet/locomotion:2.1.7
+FROM gcr.io/af-gcr/locomotion-server:add-push-to-docker-hub-latest
```

Comment:
add-push-to-docker-hub??

Reply:
still for debug, will be changed to master